session-replay/Tests/DatadogSessionReplayTests/Recorder/ViewTreeSnapshotProducer/Utilties/UIImageView+RecordingTests.swift

```diff
@@ -0,0 +1,172 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+import UIKit
+@testable import DatadogSessionReplay
+
+class UIImageViewRecordingTests: XCTestCase {
+    var frame: CGRect!
+    var imageView: UIImageView!
+
+    override func setUp() {
+        super.setUp()
+        frame = CGRect(x: 10, y: 10, width: 100, height: 100)
+        imageView = UIImageView(frame: frame)
+        if #available(iOS 13.0, *) {
```

Comment:
do we ever run tests against lower targets?

Reply:
We do, through [bitrise-nightly-unit-tests.yml](https://github.com/DataDog/dd-sdk-ios/blob/develop/tools/nightly-unit-tests/bitrise-nightly-unit-tests.yml) but this is not yet configured for SR, so not a blocker here IMO.